packages/@glimmer/component/addon/-private/base-component-manager.ts

```diff
@@ -1,5 +1,6 @@
 import { DEBUG } from '@glimmer/env';
-import { ComponentManager, ComponentCapabilities, TemplateArgs } from '@glimmer/core';
```

Comment:
TemplateArgs doesn't exist

Reply:
```
addon/-private/base-component-manager.ts:2:51 - error TS2305: Module '"../../../core/dist/commonjs"' has no exported member 'TemplateArgs'.

2 import { ComponentManager, ComponentCapabilities, TemplateArgs } from '@glimmer/core';
                                                    ~~~~~~~~~~~~
```

Reply:
Take a look at https://github.com/glimmerjs/glimmer.js/pull/334, looks like this was just missed.